packages/actor-init-sparql/test/ActorInitSparql-test.ts

```diff
@@ -621,6 +621,24 @@ describe('ActorInitSparql', () => {
             '@comunica/actor-init-sparql:baseIRI': 'myBaseIRI',
           });
       });
+
+      it('should allow optimize actors to modify the context', async() => {
+        const optimizeMediate = mediatorOptimizeQueryOperation.mediate;
```

Comment:
I would just copy-paste this method impl instead of extending it like this, to make the test more self-contained (not clear what this method does when just looking at this test).

Reply:
Done!